packages/web/src/pages/sign-up-page/components/GenrePill.tsx

```diff
@@ -0,0 +1,17 @@
+import { InputHTMLAttributes } from 'react'
+
+type GenrePillProps = InputHTMLAttributes<HTMLInputElement> & {
+  label: string
+}
+
+export const GenrePill = (props: GenrePillProps) => {
```

Comment:
is this specific to genre? maybe candidate for harmony eventually if it's a reusable pattern

Reply:
Yeah I'd love us to have this be an implementation of selectible pill. We need to figure out what a11y options we need to provide for it cause I'm using it both as a checkbox in genre select, and a radio button in select artists. Would love you help thinking this through